src/analysis/convex/cone.lean

```diff
@@ -620,17 +620,37 @@ lemma inner_dual_cone_le_inner_dual_cone (h : t ⊆ s) :
 lemma pointed_inner_dual_cone : s.inner_dual_cone.pointed :=
 λ x hx, by rw inner_zero_right
 
+lemma inner_dual_cone_singleton (x : H) :
+  ({x} : set H).inner_dual_cone = (convex_cone.positive_cone ℝ ℝ).comap (innerₛₗ x) :=
```

Comment:
Could you add a docstring for this lemma? The rhs is not what mathematicians are used to.

Reply:
What would you like it to say?

Reply:
I would write something along the lines of "the inner dual cone of a singleton is given by the preimage of the positive cone under the linear map `λ y, ⟨x,y⟩`" (I haven't checked the order in the inner product). I just think comap sounds unfamiliar to most mathematicians.

Reply:
Added, thanks; although I think mathlib is fairly decisive towards using the `comap` spelling in terms of `preimage`.

Reply:
I agree that `comap` should be prefered over `preimage` in the code. In the docstrings it is also called the pullback or inverse map (see [filter.comap](https://leanprover-community.github.io/mathlib_docs/order/filter/basic.html#filter.comap) and [submodule.comap](https://leanprover-community.github.io/mathlib_docs/linear_algebra/basic.html#submodule.comap) ) and [for the convex cone](https://leanprover-community.github.io/mathlib_docs/analysis/convex/cone.html#convex_cone.comap) it says preimage.